charts/newrelic-pixie/ci/test-values.yaml

```diff
@@ -0,0 +1,7 @@
+global:
+  licenseKey: 1234567890abcdef1234567890abcdef12345678
+  apiKey: 1234567890abcdef
+  cluster: test-cluster
+pixieClusterId: foobar
+image:
+  tag: 0.1.0-pre
```

Comment:
Is there any reason for overriding the image here? I think I'd forget to update this all the time.

Reply:
Good point. I think now that I added the `appVersion` to the chart in the most recent commits I don't need it. Let me try to remove it and see if the CI passes.